modules/nf-core/infernal/cmsearch/meta.yml

```
@@ -0,0 +1,85 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "infernal_cmsearch"
+description: Search covariance models against a sequence database
+keywords:
+  - rrna
+  - covariance model
+  - sequence search
+tools:
+  - "infernal":
+      description: "Infernal is for searching DNA sequence databases for RNA structure and sequence similarities."
+      homepage: "http://eddylab.org/infernal/Userguide.pdf"
+      documentation: "http://eddylab.org/infernal/Userguide.pdf"
+      tool_dev_url: "https://github.com/EddyRivasLab/infernal"
+      doi: "10.1093/bioinformatics/btt509"
+      licence: ["BSD-3-clause"]
+      identifier: biotools:infernal
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
```

```suggestion
          e.g. [ id:'test' ]
```
probably not applicable

---

then remove it :)